packages/core/helper-plugin/lib/src/hooks/useFocusWhenNavigate/index.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+
+const useFocusWhenNavigate = (selector = 'main', dependencies = []) => {
```

Comment:
ℹ️ allows to focus the given selector (main in this scenario) content of the page when navigating using the keyboard

Reply:
Very nice one. However, I'm not sure to understand in which case we need it. Isn't the role of the `skipToContent` button ?